src/api/http.rs

```diff
@@ -163,6 +164,17 @@ fn vdisks(bob: State<BobServer>) -> Json<Vec<VDisk>> {
     Json(vdisks)
 }
 
+#[delete("/blobs/outdated")]
+fn finalize_outdated_blobs(bob: State<BobServer>) -> Result<StatusExt, StatusExt> {
+    let backend = bob.grinder().backend();
+    runtime().block_on(backend.cleanup_outdated(1));
```

Comment:
that could take a while - so don't block

Reply:
it's different runtime, it won't block main RT